information_for_beamlines_ds/information_for_beamlines.py

```diff
@@ -143,25 +141,25 @@ def state_from_data(self, data):
         self.set_state(DevState.ON)
         self.set_status('Everything is OK.')
 
-        # ------------------
-        # Attributes methods
-        # ------------------
+    # ------------------
+    # Attributes methods
+    # ------------------
 
     def read_GeneralInfo(self):
         return self.general_info
 
     def write_GeneralInfo(self, value):
         self.general_info = value
-        # self.push_change_event('GeneralInfo')
-        # self.push_data_ready_event('GeneralInfo')
+        self.push_change_event('GeneralInfo', value)
+        self.push_data_ready_event('GeneralInfo', 0)
 
     def read_FillingPattern(self):
         return self.filling_pattern
 
     def write_FillingPattern(self, value):
         self.filling_pattern = value
-        # self.push_change_event('FillingPattern')
-        # self.push_data_ready_event('FillingPattern')
+        self.push_change_event('FillingPattern', value)
+        self.push_data_ready_event('FillingPattern', 0)
```

Comment:
Ok, now everything seems to be working. After writing something to the attributes, `change_event` is pushed.

![testevents](https://user-images.githubusercontent.com/24622343/28815703-85010b4e-76a2-11e7-9021-88828f7cbe2d.png)